security/voters.rst

```diff
@@ -323,3 +326,7 @@ security configuration:
                 'allow_if_all_abstain' => false,
             ],
         ]);
+
+.. note::
+
+    Defining a strategy may be very important when :ref:`security-allow-if`.
```

Comment:
At this spot, i don't know what `allow-if` is. Probably someone will use allow-if and THEN decide they want to change the authorization strategy and come here. In other words, I think no note is needed here.

Reply:
Alright.

Reply:
Removed.